icicle/appUtils/ntt/tests/vri2.cu

```diff
@@ -0,0 +1,228 @@
+
```

Comment:
why is this test needed in addition to the other one?

Reply:
removed